packages/atomic/cypress/e2e/facets/numeric-facet/numeric-facet.cypress.ts

```diff
@@ -178,6 +178,58 @@ describe('Numeric Facet V1 Test Suites', () => {
         });
       });
 
+      describe('with #resultsMustMatch set to "allValues"', () => {
+        function setupSelectCheckboxValue() {
+          new TestFixture()
+            .with(
+              addFacet({
+                'facet-id': 'abc',
+                field: 'objecttype',
+                'results-must-match': 'allValues',
+              })
+            )
+            .init();
+
+          selectIdleCheckboxValueAt(FacetSelectors, 0);
+        }
+
+        beforeEach(() => {
+          setupSelectCheckboxValue();
+        });
+
+        it('should include resultsMustMatch in the request', () => {
```

Comment:
'should include resultsMustMatch' -> 'should set resultsMustMatch to allValues' ?